pages/app/models/refinery/page.rb

```diff
@@ -61,7 +49,7 @@ def should_generate_new_friendly_id?
 
     has_many :parts, -> {
       scope = order('position ASC')
-      scope = scope.includes(:translations) if ::Refinery::PagePart.respond_to?(:translation_class)
+      scope = scope.includes(:translations) if ::Refinery::PagePart.respond_to?(:mobility)
```

Comment:
I think rather than use a `default_scope` on everything, you could add it here:

```ruby
scope = ::Refinery::PagePart.respond_to?(:mobility) ? i18n.includes(:translations) : all
scope = scope.order('position ASC')
scope
```